rotkehlchen/accounting/cost_basis/base.py

```diff
@@ -85,6 +89,18 @@ def serialize(self) -> Dict[str, Any]:
             'index': self.index,
         }
 
+    def __gt__(self, other: Any) -> bool:
+        if isinstance(other, AssetAcquisitionEvent):
+            return self.timestamp > other.timestamp
+
+        return self.timestamp > other
```

Comment:
here you need to be careful. When wil this happen? If this is not possible then don't do it. If it's an invalid comparison you shoudl raise a `NotImplementedError`

Reply:
A comparison takes place only when an acquisition is removed from the list.
So yeah, I'll raise a `NotImplementedError`. Thanks